### ISSUE TITLE
Fix a11y issues with banner component. 

### ISSUE THREAD

**Issue by dabowman** [Automattic/vip-cookie-banner#17]
The banner has a number of semantic a11y issues. These should be addressed and submitted as a PR to the calypso repo (if they haven't already fixed them on their end).
